plugin/restore/mysql/mysql.go

```diff
@@ -69,32 +69,33 @@ func (r *Restore) RestorePITR(ctx context.Context, fullBackup *bufio.Scanner, bi
 }
 
 // SwapPITRDatabase renames the pitr database to the target, and the original to the old database
-func (r *Restore) SwapPITRDatabase(ctx context.Context, database string, timestamp int64) error {
+// It returns the pitr and old database names after swap.
+func (r *Restore) SwapPITRDatabase(ctx context.Context, database string, suffixTs int64) (string, string, error) {
 	db, err := r.driver.GetDbConnection(ctx, "")
 	if err != nil {
-		return err
+		return "", "", err
 	}
 	txn, err := db.BeginTx(ctx, nil)
 	if err != nil {
-		return err
+		return "", "", err
 	}
 	defer txn.Rollback()
 
 	pitrOldDatabase := getSafeName(database, "old")
```

Comment:
hard to relate between getSafeName and pitrOldDatabase

Reply:
Oh, this is definitely a bug...
Should be `pitrOldDatabase := getPITROldDatabaseName(database, suffixTs)`.
We should also suffix the "old" database, in case the user does PITR multiple times on the same database.